Source/WSTagView.swift

```diff
@@ -111,6 +155,14 @@ open class WSTagView: UIView, UITextInputTraits {
         textColor = .white
         selectedColor = .gray
         selectedTextColor = .black
+        
+        imageView.frame = CGRect(x: layoutMargins.left, y: layoutMargins.top, width: imageSize.width, height: imageSize.height)
+        imageView.layer.cornerRadius = 10
```

Comment:
Why 10? With the size being (22,22) then this is just barely under the amount needed for a circle.

Reply:
Again, no reason other than that's what we have for `memo-ios`.
Should I let the user set it?